jerry-math/include/math.h

```diff
@@ -22,9 +22,9 @@ extern "C"
 #endif /* __cplusplus */
 
 /* General Constants. */
-#define INFINITY    (1.0/0.0)
-#define NAN         (0.0/0.0)
-#define HUGE_VAL    INFINITY
+#define INFINITY    ((float) (1e+300 * 1e+300)) /* 1e+300*1e+300 must overflow */
+#define NAN         (-(float) (INFINITY * 0.0F))
```

Comment:
This is confusing. Why are INFINITY and NAN "downgraded" to float? And even if they are, what's wrong with the existing computations?

Reply:
> This is confusing. Why are INFINITY and NAN "downgraded" to float? And even if they are, what's wrong with the existing computations?

msvc raise divde zero error

Reply:
I guess that this is an answer to the computation question. But why the float?

Reply:
> I guess that this is an answer to the computation question. But why the float?

Yeap, float unnecessary, removed that